data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+			label depart
+			`	You show her to her bunks. She seems satisfied, and immediately begins calling people to get things sorted. She does not seem to be the kind for casual conversation.`
+				accept
+	on accept
+		set "hr: sandsnap project"
+		log "People" "Stationkeeper Kelamursh" `Kelamursh is a high-ranking employee in an arms manufacturing company. With the concerns on Allhome, she was entrusted with the responsibility of reactivating Sandsnap Station and taking on the role of Stationkeeper.`
+
+
+mission "HRCP: Sandsnap Mercenaries"
+	landing
+	invisible
+	source "Makerplace"
+	to offer
+		has "Hai Reveal Concern Period: Reactivating Sandsnap: accepted"
+	to complete
+		never
+	npc
+		government "Hai Merchant (Human)"
+		personality escort coward opportunistic vindictive
+		fleet
+			names "hai (human)"
+			variant
+				"Grasshopper (Ion)"
+				"Grasshopper (Tracker)"
+				"Headhunter (Hai)" 2
+				"Firebird (Hai Shields)"
+				"Firebird (Hai Weapons)"
+				"Lightning Bug (Pulse)"
+				"Corvette (Hai)"
+				"Leviathan (Hai Weapons)"
+
+
+mission "Hai Reveal Concern Period: Securing Sandsnap"
+	name "Defend convoy arriving in <system>"
+	description "Keep <system> secure against pirates for the supply convoy so that they can reach <planet> safely."
+	landing
+	source "Sandsnap"
+	to offer
+		has "hr: sandsnap project"
+	on offer
+		conversation
+			`Upon arrival the Stationkeeper requests that you stay aboard your ship in case hostiles approach the station, while she suits up and goes aboard to run the access codes and start things up. You can see a selection of other Hai with various tools and equipment disembarking a few other ships in the process. Before long, the main lighting comes on in the docking section, and the team disappears inside.`
+			`	After a short while you receive a message requesting that you report to the operations center inside the main promenade. You proceed to equip your own suit and exit your ship. The station itself is well preserved, except for the lack of atmosphere. Nothing seems to have vented it, no big hole in the ceiling, but the expansive ceiling windows of the main promenade still provide a crystal clear view of the station's many decks towering above you. Exterior navigation lights are gradually flicking on up the height of the station, so that walking through feels like watching a ghost city slowly come to glittering life. Emergency floor lighting has been activated, showing you the way.`
+			`	Once through the operation center's security airlock you are able to remove your helmet. Kelamursh's subordinate, who briefed you in the offices and appears to be in charge in this area, swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`
```

Comment:
```suggestion
			`	Once through the operation center's security airlock, you are able to remove your helmet. Kelamursh's subordinate, who briefed you in the offices and appears to be in charge in this area, swiftly approaches from the other side. "Captain <last>! It is splendid you could make it here so quickly. We have begun the re-powering process, but it will take time and supplies to re-pressurize more than the docking and operations areas. These stations are not small."`
```